ja/koreader.po

```diff
@@ -1554,13 +1553,13 @@ msgid ""
 "Home folder is set to:\n"
 "%1"
 msgstr ""
-"スクリーンセイバー ディレクトリーを設定しました:\n"
+"ホームフォルダーが下記に設定されました:\n"
 "%1"
 
 #: frontend/apps/filemanager/filemanagermenu.lua:303
 #, fuzzy
```

Comment:
Btw, I don't know how you're currently editing the translations exactly? Assuming the translation is now correct, fuzzy should be removed. Normally a PO editor will do that for you, as would Weblate, so I'm guessing you might be using a text editor? There's nothing wrong with that if that's how you want to work, of course. :-)

Fuzzy either means it was automatically filled out by the computer (which is pretty dumb, but it can still be helpful to have a phrase starting with `Home folder` already in there for example), or that a human translator put in something quick to be corrected later if it's not quite right. That's similar to a fuzzy string filled out by a computer, except less dumb. ;-)

Anyway, merging this one now to prevent conflicts with Weblate.

Reply:
Hi, Frenzie, 

>  I'm guessing you might be using a text editor? 
Yes,   I am using  a plain text editor: Visual Studio code, sometimes vim. 

> Assuming the translation is now correct, fuzzy should be removed.
Thanks for pointing out the `fuzzy`  and the `weblate` convention, I 'll look into how to cooperate with them and remove the `fuzzy` indicator after refining the items.

Reply:
To remove it you can simply remove the line.

Incidentally, translations with a fuzzy attribute won't be displayed by the program.